WordPress/src/main/res/values/styles.xml

```diff
@@ -1585,27 +1589,19 @@
 
     <style name="ThemeActionSheetTitle">
         <item name="android:fontFamily">serif</item>
-        <item name="android:textColor">@color/title_color</item>
+        <item name="android:textColor">?attr/colorOnSurface</item>
         <item name="android:textSize">@dimen/theme_bottom_sheet_title_text_size</item>
         <item name="android:textStyle">bold</item>
     </style>
 
     <style name="ThemeActionSheetOptionText">
-        <item name="android:textColor">@color/title_color</item>
+        <item name="android:textColor">?attr/colorOnSurface</item>
         <item name="android:textSize">@dimen/theme_bottom_sheet_text_size</item>
     </style>
 
-    <style name="ThemeBottomSheetButtonFilled" parent="Widget.MaterialComponents.Button">
+    <style name="Widget.ModalLayoutPicker.Button.Primary.accent" parent="Widget.MaterialComponents.Button.UnelevatedButton">
```

Comment:
This is pretty minor. I would keep the previous style name (`ThemeBottomSheetButtonFilled`) or use the `Theme` in the name since the `ModalLayoutPicker` refers to another screen/feature.

Reply:
Thank you for tackling this issue with https://github.com/wordpress-mobile/WordPress-Android/pull/15029/commits/986eb25787f8b909d8813a6bd62a8e869ae93bfb @kashandanish. I really appreciate this 🙏